examples/notebooks/Splitters Tutorial.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(range(0, 7), range(7, 9), range(9, 10))"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_data,valid_data,test_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next splitter that is present in the library is the specified splitter. This splitter needs a list from the dataset where it is specified which data is for training and which is for validation and testing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loading raw samples now.\n",
+      "shard_size: 8192\n",
+      "About to start loading CSV from /home/ab/deepchem/deepchem/models/tests/user_specified_example.csv\n",
+      "Loading shard 1 of size 8192.\n",
+      "Featurizing sample 0\n",
+      "TIMING: featurizing shard 0 took 0.029 s\n",
+      "TIMING: dataset construction took 0.038 s\n",
+      "Loading dataset from disk.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from deepchem.splits.splitters import SpecifiedSplitter\n",
+    "\n",
+    "input_file=os.path.join(\"/home/ab/deepchem/deepchem/models/tests/user_specified_example.csv\")\n",
+    "\n",
+    "tasks=['log-solubility']\n",
+    "featurizer=dc.feat.CircularFingerprint(size=1024)\n",
+    "loader = dc.data.CSVLoader(tasks=tasks, smiles_field=\"smiles\",featurizer=featurizer)\n",
+    "dataset=loader.featurize(input_file)\n",
+    "\n",
+    "split_field='split'\n",
+    "\n",
+    "splitter=SpecifiedSplitter(input_file,split_field)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data,valid_data,test_data=splitter.split(dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When we split the data using the specified splitter it compares the data in each row of the `split_field` with train,test and valid and then splits the data according to the data in the `split_field`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "([0, 1, 2, 3, 4, 5], [6, 7], [8, 9])"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_data,valid_data,test_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another splitter present in the fraework is `IndiceSplitter`. This splitter takes an input of valid_indices and test_indices which are lists with the indices of validation data and test data in the dataset respectively."
```

Comment:
It would be helpful if you had a ## heading for each splitter in-between the them.